lib/services/database.go

```diff
@@ -225,6 +228,58 @@ func tagsToLabels(tags []*rds.Tag) map[string]string {
 	return labels
 }
 
+// IsRDSClusterSupported checks whether the aurora cluster is supported and logs
+// related info if not.
+func IsRDSClusterSupported(cluster *rds.DBCluster, log logrus.FieldLogger) bool {
+	switch aws.StringValue(cluster.EngineMode) {
+	// Aurora Serverless (v1 and v2) does not support IAM authentication
+	// https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless.html#aurora-serverless.limitations
+	// https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless-2.limitations.html
+	case RDSEngineModeServerless:
+		log.Infof("Aurora cluster %q is %s which doesn't support IAM authentication. Skipping.",
```

Comment:
Let's use debug level here, otherwise I think it might spam the logs on every discovery loop.
```suggestion
		log.Debugf("Aurora cluster %q is %s which doesn't support IAM authentication. Skipping.",
```